### PR TITLE
Add sqlx

### PR DESCRIPTION
**Pull request by gandaldf**
To support MultiInsert I used `NamedExec`(as per https://github.com/jmoiron/sqlx/issues/134)

### CODE REVIEW

File: benchs/sqlx.go [hunk 1]
```diff
@@ -0,0 +1,122 @@
+package benchs
+
+import (
+	"github.com/jmoiron/sqlx"
+	_ "github.com/lib/pq"
+)
+
+var sqlxdb *sqlx.DB
+
+const (
```
Comment (efectn) [inline benchs/sqlx.go:10]: I think we can use https://github.com/efectn/go-orm-benchmarks/blob/master/benchs/raw.go#L10 here.

Comment (efectn) [reply benchs/sqlx.go:10]: To make code shorter and cleaner.

**Review by efectn — CHANGES_REQUESTED** [review]
Don't forget adding sqlx to README

File: benchs/sqlx.go (hunk 1 above)
Comment (gandaldf) [reply benchs/sqlx.go:10]: > I think we can use https://github.com/efectn/go-orm-benchmarks/blob/master/benchs/raw.go#L10 here.

If you look closely the constants are slightly different (I changed the last two and added two more), so for the sake of readability I prefered let them in their own file making a little copy.

**Comment by gandaldf** [comment]
Pushed README update!

File: benchs/sqlx.go (hunk 1 above)
Comment (efectn) [reply benchs/sqlx.go:10]: Right, sorry

**Review by efectn — APPROVED** [review]

**Pull request merged**